src/ruby/pb/test/xds_client.rb

```diff
@@ -95,17 +96,45 @@ def get_client_stats(req, _call)
       end
       $watchers.delete_at($watchers.index(watcher))
     end
+    # convert results into proper proto object
+    rpcs_by_method = {}
+    watcher['rpcs_by_method'].each do | rpc_name, rpcs_by_peer |
```

Comment:
small nit: we don't have spaces before or after opening and closing `|` parameter brackets in do blocks elsewhere in the code, lets remove for consistency

Reply:
Fixed